javascripts/CarLot.js

```diff
@@ -0,0 +1,40 @@
+// main IIFE function
+//variables
+// var carContainer = document.getElementsByClassName("container");
+var CarLot = (function () {
+//The first IIFE should add a public function that loads the inventory.
+//json file and stores the inventory in a private variable.
+// It should also expose a public getter to read the array of cars (e.g. getInventory).
+  	return {
+
+  		loadInventory : function(){
+	  		var inventory = [];
+	  		var inventoryLoader = new XMLHttpRequest();
+		  		inventoryLoader.addEventListener ("load",CarLot.executeThisCodeAfterFileLoaded);
+				inventoryLoader.addEventListener("error",CarLot.executeThisCodeWhenFailure)
+				inventoryLoader.open ("GET", "inventory.json");
+				inventoryLoader.send();
+		},
+
+  		executeThisCodeAfterFileLoaded : function(){
+    		var data = JSON.parse(this.responseText);
+    		console.log("my Data is : ", data);
+    		// CarLot.resetBoarderBackground();
+    		CarLot.makeDom(data);
```

Comment:
@zoeames
i tried to fixed but i need your help 
it works only if i click on the border of the card .
thanks